src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,40 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Object[] values = new Object[MAX_SIZE];
+    private Object[] keys = new Object[MAX_SIZE];
```

Comment:
```suggestion
    private V[] values;
    private K[] keys;
```

Initialise in construstor